src/app/config/config.ts

```diff
@@ -202,6 +202,7 @@ const configureAws = async () => {
     }
   }
 }
+const secretEnv = process.env.SECRET_ENV || ''
```

Comment:
you might want to put them in `src/app/config/schema.ts` and also update `docs/DEPLOYMENT_SETUP.md` to include this environment variable

we should also set this as required, so you may want to update the PR to explicitly call out the need to add this env var to prod/staging AWS parameter store environment before we deploy